packages/client/src/raffles/types/raffles.types.ts

```diff
@@ -0,0 +1,55 @@
+export enum RaffleStatus {
+  Draft = 'Draft',
+  Ready = 'Ready',
+  AcceptingEntries = 'AcceptingEntries',
+  EntriesClosed = 'EntriesClosed',
+  WinnersSelection = 'WinnersSelection',
+  DrawClosed = 'DrawClosed',
+  Closed = 'Closed',
+}
+
+export type Raffle = {
+  id: string;
+  status: RaffleStatus;
+  title: string | null;
+  productId: number | null;
+  merchantId: number | null;
+  startDate: string | null;
+  endDate: string | null;
+  endEntriesDate: string | null;
+  drawDate: string | null;
+};
+
+export type Raffles = {
+  entries: Raffle[] | null;
+  nextToken: string | null;
+};
+
+export type RafflesNormalized = Omit<Raffles, 'entries'> & {
```

Comment:
This type should not be exported from the client package since it is only used inside redux package. Please, review and make sure no types that are used in redux only are defined and exported from the redux package instead of other packages.

Reply:
The `RafflesNormalized` type should be exported as well.

Reply:
done